app/models/event_invoice.py

```diff
@@ -12,7 +12,7 @@ class EventInvoice(SoftDeletionModel):
     __tablename__ = 'event_invoices'
 
     id = db.Column(db.Integer, primary_key=True)
-    identifier = db.Column(db.String, unique=True, default=get_new_identifier)
+    identifier = db.Column(db.String, unique=True, default = get_new_identifier(EventInvoice))
```

Comment:
This won't work, this will return the identifier, you have to create a new function which returns a function